pkg/azuredisk/controllerserver.go

```diff
@@ -429,11 +438,18 @@ func (d *Driver) ControllerPublishVolume(ctx context.Context, req *csi.Controlle
 	}
 
 	pvInfo := map[string]string{LUN: strconv.Itoa(int(lun))}
+	isOperationSucceeded = true
 	return &csi.ControllerPublishVolumeResponse{PublishContext: pvInfo}, nil
 }
 
 // ControllerUnpublishVolume detach an azure disk from a required node
 func (d *Driver) ControllerUnpublishVolume(ctx context.Context, req *csi.ControllerUnpublishVolumeRequest) (*csi.ControllerUnpublishVolumeResponse, error) {
+	mc := metrics.NewMetricContext("azuredisk_csi_driver", "controller_unpublish_volume", d.cloud.ResourceGroup, d.cloud.SubscriptionID, d.Name)
```

Comment:
use const `DriverName` intead of "azuredisk_csi_driver"

Reply:
`DriverName` is `disk.csi.azure.com`, use a const var(`azuredisk_csi_driver`) is more appropiate